LetsDoStuff.WebApi/Services/UserService.cs

```diff
@@ -36,6 +36,67 @@ public UserSettingsResponse GetUserSettings(int id)
             };
         }
 
+        public void EditUserSettings(EditUserSettingsCommand newSettings, int id)
+        {
+            using var transaction = _context.Database.BeginTransaction();
```

Comment:
Are you sure we need transactions here?

Reply:
I thought that if user wants to change his email, it can happen that we check if there is such an email in our db, see that there isn`t and after that some other user add the same email, so at the moment we save changes there is a user with email we want to add. That is why used transaction. Am I wrong?

Reply:
Not quite accurate actually. It doesn't seem like a real life scenario since an email address can be own by just one user. But in the rare case you've described a SQL exception will be thrown.
Transactions doesn't handle that situations and they are used for different purpose